controllers/apply/get-advice/form.js

```diff
@@ -0,0 +1,398 @@
+'use strict';
+const get = require('lodash/fp/get');
+const { oneLine } = require('common-tags');
+
+const Joi = require('../lib/joi-extensions');
+
+const {
+    TextField,
+    EmailField,
+    PhoneField,
+    RadioField
+} = require('../lib/field-types');
+
+const { FormModel } = require('../lib/form-model');
+
+module.exports = function({ locale = 'en', data = {} } = {}) {
+    const localise = get(locale);
+
+    const allFields = {
+        projectCountry: new RadioField({
+            name: 'projectCountry',
+            label: localise({
+                en: `What country (or countries) will your project take place in?`,
+                cy: ``
```

Comment:
Just thinking ahead for future-proofing purposes here – do you think it's worth adding a specific TODO string to these missing translations to make it easier to locate them all when it comes to translation time (rather than my gnarly regex from before)?

Reply:
I think it's OK, the number of files we need to translate will be smaller here (should mostly just be the form/field model)